cumulus/client/cirrus-executor/Cargo.toml

```diff
@@ -30,7 +30,9 @@ futures = { version = "0.3.19", features = ["compat"] }
 futures-timer = "3.0.1"
 rand = "0.8.4"
 rand_chacha = "0.3.1"
+merkletree = "0.21.0"
 parking_lot = "0.11.2"
+sha2 = "0.10.0"
```

Comment:
Check how this dependency is added to other crates to take advantage of platform-specific hardware accelerated instructions